splash/browser_tab.py

```diff
@@ -1095,6 +1121,16 @@ def log(self, message, min_level=None):
         log.msg(message, system='render')
 
 
+class ElementsStorage(QObject):
+    def __init__(self, parent):
+        self.name = str(uuid.uuid1())
+        super(ElementsStorage, self).__init__(parent)
+
+    @pyqtSlot(name="getId", result=str)
+    def get_id(self):
+        return str(uuid.uuid1())
```

Comment:
Ah, so self.name is an id of elements storage element, and get_id generates an id of an element inside this storage? This makes sense. Could you please add docstrings to make it more clear?